### PR TITLE
Services

### PR DESCRIPTION
**Pull request by alexislambda**
Wil generate a new section for services and infrastructure

-Now added: Azure, Kubernetes, AWS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -297,6 +297,60 @@ We organize activities where we share thoughts and interests with anyone who wan
 - Try to avoid timer:sleep on tests, [ktn_task:wait_for_success](https://github.com/lambdaclass/erlang-katana/blob/master/src/ktn_task.erl#L28) can be a better option. More on this [here](https://medium.com/erlang-battleground/the-missing-testing-tip-628686ebbbda).
 - Prefer maps to records.
 
+## Services and Infrastructure
+
+### ASW
```
Comment (gabrielbosio) [inline README.md:302]: Fix typo: AWS

File: README.md [hunk 2]
```diff
@@ -297,6 +297,60 @@ We organize activities where we share thoughts and interests with anyone who wan
...
+
+- Amazon Web Services (AWS) is the world’s most comprehensive and broadly adopted cloud platform, offering over 200 fully featured services from data centers globally. Millions of customers—including the fastest-growing startups, largest enterprises, and leading government agencies—are using AWS to lower costs, become more agile, and innovate faster.
+
+### ASW tools
```
Comment (gabrielbosio) [inline README.md:306]: Fix typo: AWS tools

File: README.md [hunk 3]
```diff
@@ -297,6 +297,60 @@ We organize activities where we share thoughts and interests with anyone who wan
...
+
+- IAM: It's used for managing users, groups, access policies & roles.
+- EC2: A web service that provides secure, resizable compute capacity in the cloud.
+- AMI: Amazon Machine Images is "An Amazon Machine Image (AMI) provides the information required to launch an instance".
+- ECS:  "Amazon Elastic Container Service (Amazon ECS) is a fully managed container orchestration service. Customers such as Duolingo, Samsung, GE, and Cook Pad use ECS to run their most sensitive and mission critical applications because of its security, reliability, and scalability." In simpler words, it allows you to launch containers on AWS.
+ 
+To further learn about ASW and it's practical uses go [HERE](https://github.com/bregman-arie/devops-exercises/tree/master/exercises/aws) for exercises. (AWS - EC2/IAM/Containers)
```
Comment (gabrielbosio) [inline README.md:313]: Fix typo: To further learn about AWS

File: README.md [hunk 4]
```diff
@@ -297,6 +297,60 @@ We organize activities where we share thoughts and interests with anyone who wan
...
+
+### Kubernetes
+
+Kubernetes is a portable, extensible, open-source platform for managing containerized workloads and services, that facilitates both declarative configuration and automation. It has a large, rapidly growing ecosystem. Kubernetes services, support, and tools are widely available.
+- [Getting started with kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/)
+- [Kubernete Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
```
Comment (gabrielbosio) [inline README.md:319]: Fix typo: Kubernetes Basics

File: README.md [hunk 5]
```diff
@@ -297,6 +297,60 @@ We organize activities where we share thoughts and interests with anyone who wan
...
+- [Extra exercises](https://github.com/bregman-arie/devops-exercises/tree/master/exercises/kubernetes)
+
+### Azure
+
+-[Azure FAQ]From devops-exercises repositories(https://github.com/bregman-arie/devops-exercises#azure)
```
Comment (gabrielbosio) [inline README.md:324]: Fix syntax:
Maybe this way: -[Azure FAQ](https://github.com/bregman-arie/devops-exercises#azure) From devops-exercises repositories
Or this way: -[Azure FAQ From devops-exercises repositories](https://github.com/bregman-arie/devops-exercises#azure)

File: README.md [hunk 6]
```diff
@@ -297,6 +297,60 @@ We organize activities where we share thoughts and interests with anyone who wan
...
+
+### asdf - Version manager
+
+-  [asdf](https://asdf-vm.com/guide/getting-started.html) is a version manager with the idea purpose of generating enviromental variables to choose the specific version desired. Remember that in order to be able to use the enviromental variables you need to set their path for the shell to check, you can see how to do it depending on how you installed asdf [here](https://asdf-vm.com/guide/getting-started.html#_3-install-asdf).
+
+### Node.js
+
+- [Node.js](https://nodejs.org/es/download/) is necesary for some elixir versions so is recommended to keep up to date and manage versions as needed.
+
+### Installing Erlang and Elixir
```
Comment (gabrielbosio) [inline README.md:338]: Maybe commands should be typed as monospace?
Example: `brew install erlang` or `brew install elixir`

File: README.md [hunk 7]
```diff
@@ -297,6 +297,60 @@ We organize activities where we share thoughts and interests with anyone who wan
...
+
+- On Mac distributions, you can just *brew install erlang* to get the latest installation. In case of needing an specific version you can use asdf and install with *asdf install erlang [VERSION_DESIRED]*.
+
+- Same can be achieved with Elixir through the same means *brew install elixir* for latest version, use asdf fo specific version.
+
+- For other distributions, check [here](https://learnyousomeerlang.com/introduction) for Erlang and [here](https://elixir-lang.org/install.html#distributions) for Elixir.
+
+### Installing phoenix
```
Comment (gabrielbosio) [inline README.md:346]: Capitalize Phoenix word:
Installing Phoenix
For Phoenix all dependencies...

**Review by gabrielbosio — CHANGES_REQUESTED** [review]
Some

File: README.md (hunk 1 above)
Comment (unbalancedparentheses) [reply README.md:302]: @alexislambda please fix it.

**Comment by unbalancedparentheses** [comment]
@alexislambda please solve conflicts and update the typos

File: README.md (hunk 6 above)
Comment (alexislambda) [reply README.md:365]: done

File: README.md [hunk 8]
```diff
@@ -358,15 +364,15 @@ Here are some tools that can be crucial for projects and some help with the inst
 
 ### Installing Erlang and Elixir
 
-- On Mac distributions, you can just *brew install erlang* to get the latest installation. In case of needing an specific version you can use asdf and install with *asdf install erlang [VERSION_DESIRED]*.
+- On Mac distributions, you can just <!--*brew install erlang--> to get the latest installation. In case of needing an specific version you can use asdf and install with <!--asdf install erlang [VERSION_DESIRED]-->.
```
Comment (gabrielbosio) [inline README.md:367]: I don't see the commands in the preview, maybe try with the character `

File: README.md [hunk 9]
```diff
@@ -318,31 +318,37 @@ We organize activities where we share thoughts and interests with anyone who wan
...
 Kubernetes is a portable, extensible, open-source platform for managing containerized workloads and services, that facilitates both declarative configuration and automation. It has a large, rapidly growing ecosystem. Kubernetes services, support, and tools are widely available.
 - [Getting started with kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/)
-- [Kubernete Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
+- [Kubernetes Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
 - [Extra exercises](https://github.com/bregman-arie/devops-exercises/tree/master/exercises/kubernetes)
 
 ### Azure
 
--[Azure FAQ]From devops-exercises repositories(https://github.com/bregman-arie/devops-exercises#azure)
+-[Azure FAQ From devops-exercises repositories](https://github.com/bregman-arie/devops-exercises#azure)
```
Comment (gabrielbosio) [inline README.md:351]: Add a space after the hyphen (- [Azure FAQ...) to make the unordered list

**Review by gabrielbosio — APPROVED** [review]

**Pull request merged**